ocaml/parsing/jane_syntax_parsing.ml

```diff
@@ -98,11 +98,11 @@ module Feature : sig
 
   val is_erasable : t -> bool
 end = struct
-  type t = Language_extension of Language_extension.t
+  type t = Language_extension : 'a Language_extension.t -> t
          | Builtin
 
   type error =
-    | Disabled_extension of Language_extension.t
+    | Disabled_extension of Language_extension.Exist.t
```

Comment:
`Language_extension` above is a GADT constructor; `Disabled_extension` is a non-GADT constructor taking a packed extension. Can we do the same thing for both? I don't much care which. Feel free to resolve upon addressing.